.pre-commit-config.yaml

```diff
@@ -21,8 +21,3 @@ repos:
     rev: "v2.3.2" # Use the sha or tag you want to point at
     hooks:
       - id: prettier
-  - repo: https://github.com/Yelp/detect-secrets.git
```

Comment:
Pre commit was hanging when trying to resolve "detect-secrets". Deleted while I try to solve problem.